server/core/store.go

```diff
@@ -27,6 +27,9 @@ import (
 	"go.uber.org/zap"
 )
 
+// Interval to save store meta (including heartbeat ts) to etcd.
+const storePersistInterval = 5 * time.Minute
```

Comment:
Do we need to let users know the exact downtime may not as they expected under this case?

Reply:
yes, I can update the document.